Libraries/LibWeb/HTML/HTMLQuoteElement.idl

```diff
@@ -6,6 +6,6 @@ interface HTMLQuoteElement : HTMLElement {
 
     [HTMLConstructor] constructor();
 
-    [CEReactions, Reflect] attribute USVString cite;
+    [CEReactions] attribute USVString cite;
```

Comment:
I think simply changing this to `[CEReactions, Reflect, URL]` would work here.

Reply:
Thanks for the advice, didn't know that was possible.